# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bochs/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur bochs-2.2.6-orig/iodev/hdimage.h bochs-2.2.6/iodev/hdimage.h
--- bochs-2.2.6-orig/iodev/hdimage.h	2005-11-06 11:07:01.000000000 +0000
+++ bochs-2.2.6/iodev/hdimage.h	2006-04-02 10:32:32.095216500 +0000
@@ -272,16 +272,8 @@
  off_t total_size;
 
  void panic(const char * message);
- off_t
-#ifndef PARANOID
-       sparse_image_t::
-#endif
-                       get_physical_offset();
- void
-#ifndef PARANOID
-       sparse_image_t::
-#endif
-                       set_virtual_page(Bit32u new_virtual_page);
+ off_t get_physical_offset();
+ void set_virtual_page(Bit32u new_virtual_page);
  void read_header();
  ssize_t read_page_fragment(Bit32u read_virtual_page, Bit32u read_page_offset, size_t read_size, void * buf);
 
